docker/entrypoint.sh

```diff
@@ -12,9 +12,16 @@ trap catchall INT EXIT TERM
 
 # Stage 1, as root
 if [ $EUID = 0 ] ; then
-	# Align local temboard user with remote postgres user.
+	# Fix UID and GID
+	groupmod -g $(stat -c "%g" /var/run/docker.sock) docker
 	usermod -u $(stat -c "%u" /var/lib/postgresql/data) temboard
-	groupmod -g $(stat -c "%g" /var/lib/postgresql/data) temboard
+	if ! group=$(stat -c "%G" /var/lib/postgresql/data) ; then
```

Comment:
I got a case where `stat -c "%G" /var/lib/postgresql/data` returns "UNKNOWN" which leads to an error when trying to add the user temboard to this group.

```
++ stat -c %G /var/lib/postgresql/data
+ group=UNKNOWN
+ groups temboard
+ grep -q UNKNOWN
+ adduser temboard UNKNOWN
adduser: The group `UNKNOWN' does not exist.
```